fs/dtype_test.go

```diff
@@ -0,0 +1,18 @@
+package fs
+
+import (
+	"testing"
+
+	"github.com/containerd/containerd/testutil"
+)
+
+func TestRequiresRootNOP(t *testing.T) {
+
+	// This is a dummy test case that exist to call
```

Comment:
@epilatow no, your analysis looks correct to me. Thanks!

What do you think about moving the flag registration to `client_test.go`?

Either way, the dummy test will do for now.